pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -1270,3 +1270,12 @@ func (c *K8sOrchestrator) GetVolumeAttachment(ctx context.Context, volumeId stri
 	}
 	return volumeAttachment, nil
 }
+
+// GetAllVolumes returns list of volumes in a bound state
+func (c *K8sOrchestrator) GetAllVolumes() []string {
```

Comment:
This function is needed only for WCP ? Or will this be needed for Vanilla k8s clusters as well ?
Asking this because, this function doesn't return volumeIds for VCP-CSI migrated volumes

Reply:
This is used only by wcp.